lib/streams.gi

```diff
@@ -1221,27 +1221,26 @@ InstallMethod( SetPrintFormattingStatus, "output text file",
     fi;
 end);
 
-##  formatting status for stdout
-GAPInfo.FormattingStatusStdout := true;
+##  formatting status for stdout or current output
 InstallOtherMethod( PrintFormattingStatus, "for stdout", [IsString],
 function(str)
-  if str <> "*stdout*" then
-    Error("Only the string \"*stdout*\" is recognized by this method.");
+  if str = "*stdout*" then
+    return PRINT_FORMATTING_STDOUT();
+  elif str = "*current*" then
+    return PRINT_FORMATTING_CURRENT();
```

Comment:
Since we were discussing also handling `*errout*`, perhaps there should be a single pair of kernel function `(SET_)PRINT_FORMATTING_STATUS(name, ...)` which handles `*stdout*`/`*errout*`/`*current*` plus future additions?

Ah well, I guess it's not really important whether we do it one way or another *shrug*